HLTrigger/Muon/src/HLTScoutingMuonProducer.cc

```diff
@@ -145,7 +145,8 @@ void HLTScoutingMuonProducer::produce(edm::StreamID sid, edm::Event & iEvent,
       
       double hcalisopf=-1.0;
       if ( HcalPFClusterIsoMap.isValid()) { hcalisopf = (*HcalPFClusterIsoMap)[muonRef]; }
-      
+ 
+      vtxInd.reserve(vtxMuPair.size());  
```

Comment:
hi @swagata87 - please move the vtxInd declaration to here - then the clear/reserve calls are not needed